video/out/hwdec/hwdec_cuda_vk.c

```diff
@@ -229,6 +289,27 @@ static bool cuda_ext_vk_wait(const struct ra_hwdec_mapper *mapper, int n)
     int ret;
     struct ext_vk *evk = p->ext[n];
 
+#if HAVE_LIBPLACEBO_NEXT
+    evk->vk_sem.value += 1;
+    ret = pl_vulkan_hold_ex(ra_pl_get(mapper->ra), pl_vulkan_hold_params(
+        .tex = evk->pltex,
+        .layout = VK_IMAGE_LAYOUT_GENERAL,
+        .qf = VK_QUEUE_FAMILY_EXTERNAL,
+        .semaphore = evk->vk_sem,
+    ));
+    if (!ret)
+        return false;
```

Comment:
In this case you probably don't want to increment `vk_sem.value`? Though I guess it doesn't really matter as we probably lost the device.

Reply:
I figured that because if you fail to signal, you skip the wait, and you always increment before signalling, it doesn't matter if the previous value was never actually signalled. On top of that, as you note, there's basically no failure path here that allows us to make meaningful forward progress.